packages/components/src/range-control/index.js

```diff
@@ -30,13 +30,16 @@ function RangeControl( {
 	min,
 	max,
 	setState,
+	initialValue,
 	...props
 } ) {
 	const id = `inspector-range-control-${ instanceId }`;
 	const currentInputValue = currentInput === null ? value : currentInput;
+	const initialInputValue = ! initialValue && min <= initialValue && max >= initialValue ?
```

Comment:
When I set the initial value to 11 for the RSS block, it goes out of min/max range. There must be some error in this condition.

Reply:
Fixed! Will push this once I finish the other changes :)